config/initializers/rabl_init.rb

```diff
@@ -56,7 +56,8 @@ def data_name(data_token)
   # config.include_child_root = true
   # config.enable_json_callbacks = false
   # config.xml_options = { :dasherize  => true, :skip_types => false }
-  config.view_paths = [Rails.root.join('app', 'views')]
+  config.view_paths ||= []
```

Comment:
Actually, I think we can leave this out as view_paths always seems to default to `[]`.